src/Silindir.java

```diff
@@ -0,0 +1,30 @@
+public class Silindir extends Daire {
+    double yukseklik;
```

Comment:
yükseklik degerine nerden ulasmak istediğimizi belirtmek için ulasım belirteci belirlememişsiniz